src/commands/react_native/xcode.rs

```diff
@@ -324,8 +324,14 @@ pub fn wrap_call() -> Result<()> {
     let mut args: Vec<_> = env::args().skip(1).collect();
     let mut bundle_path = None;
     let mut sourcemap_path = None;
+    let bundle_command = env::var("SENTRY_RN_BUNDLE_COMMAND");
 
-    if args.len() > 1 && (args[1] == "bundle" || args[1] == "ram-bundle") {
+    // bundle and ram-bundle are React Native CLI commands
+    // export:embed is an Expo CLI command (drop in replacement for bundle)
+    // if bundle_command is set, ignore the default values
+    if args.len() > 1
+      && ((bundle_command.is_err() && (args[1] == "bundle" || args[1] == "ram-bundle" || args[1] == "export:embed"))
+        || (bundle_command.is_ok() && args[1] == bundle_command.unwrap())) {
```

Comment:
Does this mean that you have to pass the command in both the environment variable and the first parameter?

Reply:
We replace the NodeJS binary with Sentry CLI. When RN or Expo CLI is executed by NodeJS, we look at the parameters and check if the bundle command was called.

The command in the params is passed in by the RN/Expo tooling. We check if it's one of the known bundle commands or if the `SENTRY_RN_BUNDLE_COMMAND` is set we check if the passed command in the args is equal to the env.

Reply:
Right, thanks for the explanation!